platform/ios/uitest/ios-tests.xcodeproj/project.pbxproj

```diff
@@ -284,6 +284,9 @@
 				LastUpgradeCheck = 0700;
 				ORGANIZATIONNAME = Mapbox;
 				TargetAttributes = {
+					DD043322196DB9BC00E6F39D = {
+						DevelopmentTeam = GJZR2MEM28;
+					};
```

Comment:
Sounds good @incanus - yeah, if those are the only options we have atm then we can come up with some [pre-commit script hook](http://githooks.com/) in git that automatically removes these changes.